test/e2e/resize_nodes.go

```diff
@@ -372,18 +410,19 @@ var _ = Describe("Nodes", func() {
 					break
 				}
 			}
-			Logf("Setting network partition on %s", node.Name)
+			By(fmt.Sprintf("cause network partition on node %s", node.Name))
+
 			dropCmd := fmt.Sprintf("sudo iptables -I OUTPUT 1 -d %s -j DROP", testContext.CloudConfig.MasterName)
```

Comment:
Does this actually partition the network or just drop traffic from (to?) the master?

Can you add a comment stating the exact effects on network traffic this command has?

Sorry, I know you didn't add this in this PR but without this knowledge it's difficult to review this.